Can't  compile  lsc  with  maven  3.8.4

Using  fedora core 36  and launching 
`
mvn -Popendj clean package
`

fails with

```
[ERROR] Failed to execute goal org.reflections:reflections-maven:0.9.9-RC1:reflections (default) on project lsc-core: Execution default of goal org.reflections:reflections-maven:0.9.9-RC1:reflections failed: Plugin org.reflections:reflections-maven:0.9.9-RC1 or one of its dependencies could not be resolved: Failed to collect dependencies at org.reflections:reflections-maven:jar:0.9.9-RC1 -> org.jfrog.jade.plugins.common:jade-plugin-common:jar:1.3.8: Failed to read artifact descriptor for org.jfrog.jade.plugins.common:jade-plugin-common:jar:1.3.8: Could not transfer artifact org.jfrog.jade.plugins.common:jade-plugin-common:pom:1.3.8 from/to maven-default-http-blocker (http://0.0.0.0/): Blocked mirror for repositories: [jfrog-plugins (http://repo.jfrog.org/artifactory/plugins-releases, default, releases+snapshots)] -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/PluginResolutionException
```

maven  version  

```
 mvn --version
Apache Maven 3.8.4 (Red Hat 3.8.4-3)
Maven home: /usr/share/maven
Java version: 11.0.16.1, vendor: Red Hat, Inc., runtime: /usr/lib/jvm/java-11-openjdk-11.0.16.1.1-1.fc36.x86_64
Default locale: fr_FR, platform encoding: UTF-8
OS name: "linux", version: "5.19.9-200.fc36.x86_64", arch: "amd64", family: "unix"
```

related  discussion  here : https://mail.ow2.org/wws/arc/lsc-users/2022-05/msg00000.html

--- a/pom.xml
+++ b/pom.xml
@@ -11,7 +11,7 @@
 		simplifies synchronizations between relation databases and LDAP
 		directories.
 	</description>
-	<url>http://lsc-project.org/</url>
+	<url>https://lsc-project.org/</url>
 	<issueManagement>
 		<system>github</system>
 		<url>https://github.com/lsc-project/lsc/issues</url>
@@ -51,7 +51,7 @@
 	<licenses>
 		<license>
 			<name>BSD</name>
-			<url>http://www.opensource.org/licenses/bsd-license.php</url>
+			<url>https://www.opensource.org/licenses/bsd-license.php</url>
 		</license>
 	</licenses>
 
@@ -63,7 +63,7 @@
 
 	<organization>
 		<name>LSC project</name>
-		<url>http://lsc-project.org/</url>
+		<url>https://lsc-project.org/</url>
 	</organization>
 
 	<properties>
@@ -483,26 +483,26 @@
 	<repositories>
 		<repository>
 			<id>java.net Maven 2</id>
-			<url>http://download.java.net/maven/2</url>
+			<url>https://download.java.net/maven/2</url>
 		</repository>
 		<repository>
                         <id>forgerock-community-repository</id>
                         <name>ForgeRock CommunityRepository</name>
-                        <url>http://maven.forgerock.org/repo/community</url>
+                        <url>https://maven.forgerock.org/repo/community</url>
                 </repository>
 		<repository>
 			<id>oracleReleases</id>
 			<name>Oracle Released Java Packages</name>
-			<url>http://download.oracle.com/maven</url>
+			<url>https://download.oracle.com/maven</url>
 		</repository>
 		<repository>
 			<id>reflections-repo</id>
 			<name>Reflections Maven2 Repository</name>
-			<url>http://reflections.googlecode.com/svn/repo</url>
+			<url>https://reflections.googlecode.com/svn/repo</url>
 		</repository>
 		<repository>
 			<id>lsc-site</id>
-			<url>http://lsc-project.org/maven</url>
+			<url>https://lsc-project.org/maven</url>
 			<releases>
 				<enabled>true</enabled>
 			</releases>
@@ -517,7 +517,12 @@
 		<pluginRepository>
 			<id>reflections-repo</id>
 			<name>Reflections Maven2 Repository</name>
-			<url>http://reflections.googlecode.com/svn/repo</url>
+			<url>https://reflections.googlecode.com/svn/repo</url>
+		</pluginRepository>
+                <pluginRepository>
+                  <id>org.jfrog.jade.plugins.common</id>
+                  <name>openmindonline-jade-plugin</name>
+                  <url>https://repository.openmindonline.it</url>
 		</pluginRepository>
 	</pluginRepositories>
 

